src/django_components/templatetags/component_tags.py

```diff
@@ -674,43 +685,3 @@ def _fix_nested_tags(parser: Parser, block_token: Token) -> None:
 
             expects_text = True
             continue
-
-
-class ParsedSlotTag(NamedTuple):
-    name: str
-
-
-def _parse_slot_args(
```

Comment:
`_parse_slot_args` and `_parse_provide_args` were moved to `SlotNode.resolveKwargs` and `ProvideNode.resolveKwargs` respectively.